oak_containers_launcher/src/server.rs

```diff
@@ -170,6 +170,21 @@ impl Launcher for LauncherServerImplementation {
             .map_err(|_err| tonic::Status::internal(format!("couldn't send notification")))?;
         Ok(tonic::Response::new(()))
     }
+
+    async fn log(&self, request: Request<LogRequest>) -> Result<Response<()>, tonic::Status> {
```

Comment:
Optional: It feels like client streaming is more appropriate for logging than a bunch of unary requests with empty responses. But, I think it will be more complicated to manage the open stream, so perhaps the added complexity is not worth it.

Reply:
+1 for streaming -- I don't expect it to be substantially more complicated.